taxonomies/packaging_materials.txt

```diff
@@ -112,211 +203,456 @@ fr:Plastiques mixtes
+description:pt:Garrafas PET convencionais que foram tornadas opacas pela adição de certos pigmentos de cor, como dióxido de titânio para branco, para ter uma barreira à luz para proteger líquidos, como leite. Este material requer um processo específico de reciclagem.
 
 <en:Plastic
 en:PE - Polyethylene, Polyethylene
+bg:PE - Полиетилен, Полиетилен
+ca:PE - polietilè, polietilè
+cs:PE - polyethylen, polyethylen
+da:PE - polyethylen, polyethylen
 de:PE - Polyethylen, Polyethylen
+el:PE - Πολυαιθυλένιο, Πολυαιθυλένιο
+es:PE - polietileno, polietileno
+et:PE - polüeteen, polüeteen
+fi:PE - polyeteeni, polyeteeni
 fr:PE - Polyéthylène, Polyéthylène
-hr:PE - Polietilen
+hr:PE - Polietilen, Polietilen
+hu:PE - polietilén, polietilén
+is:PE - Fjöletýlen, Fjöletýlen
+it:PE - polietilene, polietilene
+lt:PE - Polietilenas, Polietilenas
+lv:PE - Polietilēns, Polietilēns
+nb:PE - polyetenpolyeten, polyeten
+nl:PE - polyetheen, polyetheen
+nn:PE - polyetylen, polyetylen
+pl:PE - polietylen, polietylen
+pt:PE - polietileno, polietileno
+ro:PE - Polietilenă, Polietilenă
+ru:PE - полиэтилен, полиэтилен
+sk:PE - Polyetylén, Polyetylén
+sl:PE - Polieten, Polieten
+sq:PE - Polietileni, Polietileni
+sv:PE - polyeten, polyeten
+uk:PE - поліетилен, поліетилен
 xx:PE, 07 PE, PE 07, 07PE, PE07, 7 PE, PE 7, 7PE, PE7
 wikidata:en:Q143429
+description:de:Polyethylen ist der am häufigsten verwendete Kunststoff. Es ist gut zu recyceln, weil es eingeschmolzen werden kann. Es kommt als HDPE (High Density) und als LDPE (Low Density) vor.
+description:en:Polyethylene is the most commonly used plastic. It is good to recycle because it can be melted down. It occurs as HDPE (high density) and as LDPE (low density).description:es:El polietileno es el plástico más utilizado. Es bueno reciclar porque se puede derretir. Se presenta como HDPE (alta densidad) y como LDPE (baja densidad).
+description:fr:Le polyéthylène est le plastique le plus couramment utilisé. Il est bon de recycler car il peut être fondu. Il se présente sous forme de HDPE (haute densité) et de LDPE (basse densité).
+description:it:Il polietilene è la plastica più comunemente usata. È bene riciclare perché può essere sciolto. Si presenta come HDPE (alta densità) e come LDPE (bassa densità).
+description:nl:Polyetheen is de meest gebruikte plastic. Het is geod te recycleren, omdat het kan worden omgesmolten. Het komt voor als HDPE (hoge dichtheid) en als LDPE (lage dichtheid).
+description:pt:O polietileno é o plástico mais utilizado. É bom reciclar porque pode ser derretido. Ocorre como HDPE (alta densidade) e como LDPE (baixa densidade).
 
 <xx:PE
-en:Biobased PE
-de:Biobasiertes PE
-fr:PE biosourcé
-hu:Biobázisú PE
-pt:Bioplástico PE
-xx:Bio-PE
+en:bioPE - Biobased 
```

Comment:
same here, "Biobased PE" should be  added as a synonym

Reply:
done